sphinx-mode.el

```diff
@@ -123,49 +122,54 @@ If BUFFER is not given use the `current-buffer'."
     (find-file (plist-get target :file))
     (goto-char (plist-get target :point))))
 
-(defun sphinx-compile-find-makefile-dir () ; adapted from rst-compile-find-conf
-  "Look for the Sphinx Makefile in parents of the current path and returns the directory which contains it as a string."
+(defun sphinx-compile ()
+  "Runs 'make' in project root directory, prompting user for target format. Run 'make help' to see list of target formats."
   (interactive)
-  (let ((file-name "Makefile")
-        (buffer-file (buffer-file-name)))
-    ;; Move up in the dir hierarchy till we find Makefile.
-    (let* ((dir (file-name-directory buffer-file))
-	   (prevdir nil))
-      (while (and (or (not (string= dir prevdir))
-		      (setq dir nil)
-		      nil)
-                  (not (file-exists-p (concat dir file-name))))
-        ;; Move up to the parent dir and try again.
-	(setq prevdir dir)
-        (setq dir (expand-file-name (file-name-directory
-                                     (directory-file-name
-				      (file-name-directory dir))))))
-      (or (and dir (concat dir "")) nil))))
-
-;; TODO: replace this with sphinx-compile and sphinx-compile-and-view
-(defun sphinx-view-build-html ()
-  "Runs 'make html' in project root directory and then views compiled HTML of current source file."
+    (let
```

Comment:
The default lisp style is to keep all the closing parens on the same line.  The one opening the `let` is also together, so 

```
(let ((foo bar)
      (... ...)))
```